frontend/src/app/game/mitigations-control/controls/mitigation-scale.component.ts

```diff
@@ -2,7 +2,7 @@ import {Component, Input, OnChanges, SimpleChanges, ViewChild} from '@angular/co
 import {ControlValueAccessor, NG_VALUE_ACCESSOR} from '@angular/forms';
 import {MatSlider} from '@angular/material/slider';
 
-type Level = [value: any, label: string];
+export type Level = [value: any, label: string];
```

Comment:
What is this syntax? `[value: any, label: string];` My IDE screams a lot! :D

Reply:
It's TS labeled tuple: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html
Introduced in TS 4